### PR TITLE
added input component

### PR DESCRIPTION
**Pull request by jshin029**
Attached three versions of input component
- type="text"
- type="password"
- placeholder="Search..."

Screen Shot 2020-04-12 at 4 13 24 PM
Screen Shot 2020-04-12 at 4 13 42 PM
Screen Shot 2020-04-12 at 4 13 55 PM

### CODE REVIEW

File: src/components/Input/Input.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.canopy__input {
+  font-weight: 500;
```
Comment (chefong) [inline src/components/Input/Input.css:2]: This should be `font-weight: 700`

File: src/components/Input/Input.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  border-radius: 4px;
+  font-family: "Geomanist";
+  border: 0px;
+  padding: 0.75%;
+  background-color: #F4F4F4;
+}
+
+.canopy__input:hover {
+  cursor: text;
+}
```
Comment (chefong) [inline src/components/Input/Input.css:14]: I don't think this is needed

File: src/components/Input/Input.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.canopy__input {
+  font-weight: 500;
+  display: block;
+  margin: 0 auto;
+  border-radius: 4px;
+  font-family: "Geomanist";
+  border: 0px;
+  padding: 0.75%;
```
Comment (chefong) [inline src/components/Input/Input.css:8]: Padding should be 16px around (according to Figma)

File: src/components/Input/Input.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { types, selectStyle } from './constants';
```
Comment (chefong) [inline src/components/Input/Input.js:3]: `selectStyle` isn't needed anymore here

File: src/components/Input/Input.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.canopy__input {
```
Comment (chefong) [inline src/components/Input/Input.css:1]: * The `placeholder` text color should be set to the `dark-alt` hex color
* The input color should be the `dark` hex color

Both can be found in the design token readme
https://github.com/Umbrlla/design-tokens

**Review by chefong — CHANGES_REQUESTED** [review]
Some comments, overall almost good to go

**Comment by jshin029** [comment]
changes have been made

File: src/components/Input/Input.css [hunk 6]
```diff
@@ -1,16 +1,19 @@
...
-  font-weight: 500;
+  font-weight: 700;
+  font-size: 16px;
   display: block;
   margin: 0 auto;
   border-radius: 4px;
   font-family: "Geomanist";
   border: 0px;
-  padding: 0.75%;
+  padding: 8px;
```
Comment (chefong) [inline src/components/Input/Input.css:9]: Sorry can we make this 16 px?

File: src/components/Input/Input.css [hunk 7]
```diff
@@ -1,16 +1,19 @@
 .canopy__input {
-  font-weight: 500;
+  font-weight: 700;
```
Comment (chefong) [inline src/components/Input/Input.css:2]: My bad i think this should be 400

File: src/components/Input/Input.css [hunk 8]
```diff
@@ -1,16 +1,19 @@
...
+  font-size: 16px;
   display: block;
   margin: 0 auto;
   border-radius: 4px;
   font-family: "Geomanist";
   border: 0px;
-  padding: 0.75%;
+  padding: 8px;
   background-color: #F4F4F4;
+  text-color: #403B3B;
```
Comment (chefong) [inline src/components/Input/Input.css:11]: the property should be `color` not `text-color`

**Review by chefong — CHANGES_REQUESTED** [review]
Sorry some small things again

**Review by chefong — APPROVED** [review]
lgtm

**Pull request merged**